### PR TITLE
refactor: moves explorer to device detail page

### PR DESCRIPTION
**Pull request by matt-primrose**
## PR Checklist

- [ ] Unit Tests have been added for new changes
- [ ] API tests have been updated if applicable
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with Apache 2.0 and clearly outlined the added dependency.

### CODE REVIEW

**Review by rsdmike — CHANGES_REQUESTED** [review]
missing condition for cloudmode

File: src/app/explorer/explorer.component.ts [hunk 1]
```diff
@@ -62,7 +65,8 @@ export class ExplorerComponent implements OnInit {
         map((value) => this._filter(value ?? ''))
       )
       this.activatedRoute.params.subscribe((params) => {
-        this.devicesService.executeExplorerCall(params.id as string, this.selectedWsmanOperation).subscribe((data) => {
+        this.deviceId = params.id
+        this.devicesService.executeExplorerCall(this.deviceId, this.selectedWsmanOperation).subscribe((data) => {
```
Comment (rsdmike) [inline src/app/explorer/explorer.component.ts:69]: missing error handler snackbar message

File: src/app/explorer/explorer.component.ts [hunk 2]
```diff
@@ -77,10 +81,8 @@ export class ExplorerComponent implements OnInit {
 
   inputChanged(value: any): void {
     this.selectedWsmanOperation = value
-    this.activatedRoute.params.subscribe((params) => {
-      this.devicesService.executeExplorerCall(params.id as string, this.selectedWsmanOperation).subscribe((data) => {
-        this.XMLData = data
-      })
+    this.devicesService.executeExplorerCall(this.deviceId, this.selectedWsmanOperation).subscribe((data) => {
```
Comment (rsdmike) [inline src/app/explorer/explorer.component.ts:84]: missing error handler snackbar message

File: src/app/explorer/explorer.component.ts (hunk 1 above)
Comment (matt-primrose) [reply src/app/explorer/explorer.component.ts:69]: added error handler

File: src/app/explorer/explorer.component.ts (hunk 2 above)
Comment (matt-primrose) [reply src/app/explorer/explorer.component.ts:84]: added error handler

**Review by rsdmike — APPROVED** [review]

**Pull request merged**